module_utils/network_lsr/argument_validator.py

```diff
@@ -92,12 +92,12 @@ class ArgValidator:
     def __init__(self, name=None, required=False, default_value=None):
         self.name = name
         self.required = required
-        self.default_value = default_value
+        self._default_value = default_value
 
     def get_default_value(self):
```

Comment:
Since you are changing this, it seems to be more pythonic to me to make `default_value` a property, then.

Reply:
That `@property` makes attributes visually indistinguishable from a method call, seems undesirable.

Granted, in python the distinction between what looks like an access to an attribute, versus an underlying function call is probably subtle. Important is that `self.get_default_value()` makes it more obvious that something gets computed and that it returns a new value. In fact, it's important that if this function returns either an immutable value or a clone that is owned by the user (that is, `self._default_value = list()` would be wrong, it should be `self._default_value = list`).

Also, again, the term "default_value" is for different purposes. I want to be able to search the source for the callers of this function using a unique name.